oracle/README.md

```diff
@@ -321,6 +322,43 @@ Verifies the database is available and accepting connections.
 
 ## Troubleshooting
 
+### Common Problems
+#### Oracle Instant Client
+- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
+
+##### Linux
+- See further installation documentation on [Oracle][15].
+
+##### Windows
+- Verify the Microsoft Visual Studio <YEAR> Redistributable requirement is met for your version, see [Windows downloads page][16] for more details.
+- Ensure the `PATH` environment variable contains `C:\oracle\instantclient_19_9`. 
+
+    Otherwise, navigate to Control Panel -> Settings -> Advanced System Settings -> Environment Variables and
```

Comment:
IIRC, the installation does not update the PATH automatically.

Reply:
Created a windows installation step